usr/bin/revpi-btuart

```diff
@@ -4,5 +4,15 @@
 HCIATTACH=/usr/bin/hciattach
 TTYDEVNAME="$1"
 
+# RevPi Flat: The BT_REG_ON signal is connected to the GPIO11 of the expander.
+GPIOCHIP=gpiochip3
+BT_REG_ON_GPIO=11
+
+# Reset the bt module in RevPi Flat
+gpioset ${GPIOCHIP} ${BT_REG_ON_GPIO}=0
+sleep 0.1
+gpioset ${GPIOCHIP} ${BT_REG_ON_GPIO}=1
```

Comment:
The reference of the shell variable should be in the same style, $VAR, or ${VAR}

Reply:
ack